src/context/WalletProvider/KeepKey/PinModal.tsx

```diff
@@ -18,10 +18,12 @@ export const PinModal = () => {
 
   return (
     <>
-      <ModalHeader>Enter Your Pin</ModalHeader>
+      <ModalHeader>
+        <Text translation={'walletProvider.pinModal.header'} />
+      </ModalHeader>
       <ModalBody>
         <RawText color='gray.500'>
-          Use PIN layout shown on your device to find the location to press on this pin pad.
+          <Text translation={'walletProvider.pinModal.body'} />
```

Comment:
There is no need to have the RawText component wrapping the Text component we should just add the RawText props to the Text component props and get rid of the RawText component.
```suggestion
          <Text color='gray.500' translation={'walletProvider.pinModal.body'} />
```